server/routerlicious/packages/lambdas/src/scribe/summaryWriter.ts

```diff
@@ -639,7 +639,7 @@ export class SummaryWriter implements ISummaryWriter {
                     break;
                 }
                 default: {
-                    throw new Error(`Unexpected TreeEntry type: ${treeEntry.type} when converting ITreeEntry.`);
```

Comment:
Is this intentional? I see `treeEntry.type` inspected in the switch statement..

Reply:
yes, I was getting an error as before, there was another type that we were not dealing with (Commit) - now, there are only 3 types, and they are all dealt with on the switch. "Property 'type' does not exist on type 'never'.ts(2339)"